### PR TITLE
DnD empty groups

### PR DESCRIPTION
**Pull request by ptbrowne**
Add empty component for when there are no groups in table chart options. Still draft, need input from @AnninaWalker.

### CODE REVIEW

File: app/themes/federal.tsx [hunk 1]
```diff
@@ -165,18 +165,18 @@ theme.typography = merge(theme.typography, {
     fontWeight: "bold",
   }),
   body1: createTypographyVariant(theme, {
-    fontSize: [14, 16],
-    lineHeight: [22, 24],
+    fontSize: 14,
+    lineHeight: [24],
     fontWeight: "regular",
   }),
```
Comment (bprusinowski) [inline app/themes/federal.tsx:171]: I think it should be fontSize: [16] here, no?

Comment (ptbrowne) [reply app/themes/federal.tsx]: I am keeping this for another PR, I need to see how to see this in the sketch file.

Comment (ptbrowne) [reply app/themes/federal.tsx]: I extracted the commit

**Review by bprusinowski — APPROVED** [review]
👌

**Pull request merged**